src/Application/GameStarter.cpp

```diff
@@ -137,7 +139,7 @@ GameStarter::GameStarter(GameStarterOptions options): _options(std::move(options
     if (_nuklear) {
         _application->installComponent(std::make_unique<NuklearEventHandler>());
         _nuklear->addInitLuaFile("init.lua");
-        _nuklear->addInitLuaLibs(GameLuaBindings::init);
+        _nuklear->addInitLuaLibs([this](auto &&luaState) { _gameLuaBindings->init(luaState); });
```

Comment:
Let's spell out the `lua_State *` type explicitly. Otherwise `auto &&` implies perfect forwarding, which cannot be done with this syntax (`std::forward<decltype(luaState)>` is just ugly).